src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,127 @@
+package mate.jdbc.dao;
+
+import java.sql.*;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Optional;
+
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+import mate.jdbc.util.DataProcessingException;
+import mate.jdbc.dao.interfaces.ManufacturerDao;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    private Connection connection;
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        connection = ConnectionUtil.getConnection();
+        String queryCreate = "INSERT INTO manufacturers (`name`, `country`) VALUES (?, ?)";
+        try (PreparedStatement statement = connection.prepareStatement(queryCreate,
+                Statement.RETURN_GENERATED_KEYS)) {
+            statement.setString(1, manufacturer.getName());
+            statement.setString(2, manufacturer.getCountry());
+            statement.executeUpdate();
+            ResultSet resultSet = statement.getGeneratedKeys();
+            if (resultSet.next()) {
+                Long insertedId = resultSet.getLong(1);
+                manufacturer.setId(insertedId);
+            }
+            manufacturer.setDeleted(false);
+            connection.close();
+        } catch (SQLException e) {
+            throw new DataProcessingException("error in create method "
+                    + manufacturer.getName()
+                    + " and "
+                    + manufacturer.getCountry(), e);
```

Comment:
Override toString() if needed
```suggestion
            throw new DataProcessingException("Can't create manufacturer: "
                    + manufacturer, e);
```